src/components/RestaurantDetails/RestaurantDetails.js

```
@@ -0,0 +1,74 @@
+/* eslint-disable jsx-a11y/img-redundant-alt */
+//Libraries
+import React from 'react';
+
+//Helpers
+import fetchData from '../../api/fetchData';
+import restaurantDetailsReader from './readers/restaurantDetailsReader';
+
+//Constants
+import { INITIAL_STATE } from './constants/general';
+
+//Components
+import Offers from './components/Offers';
+import Filters from './components/Filters';
+import Info from './components/Info';
+import Loader from './components/Loader';
+import ErrorHandler from './components/ErrorHandler';
+//CSS
+import './RestaurantDetails.css';
+
+class RestaurantDetails extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = INITIAL_STATE;
+    this.fetchRestaurantDetails = this.fetchRestaurantDetails.bind(this);
+  }
+  fetchRestaurantDetails() {
+    fetchData('restaurantDetails.json')
+      .then((response) => {
+        this.setState({
+          restaurantDetails: response,
+        });
+      })
+      .catch((error) => {
+        console.log(error);
```

instead of logging error throw it for now, or have a placeholder error component